.circleci/config.yml

```diff
@@ -549,7 +548,7 @@ workflows:
       - simulate_sequence:
           name: simulate_sequence_eth
           network: "eth"
-          tasks: ""
+          tasks: "" # tasks: "011-unpause-superchainConfig"
```

Comment:
this looks wrong?

Reply:
anyway, I have fixed the CI and this would require to merge the activation before!